src/sql/parts/dashboard/widgets/insights/insightsWidget.component.ts

```diff
@@ -300,28 +301,15 @@ export class InsightsWidget extends DashboardWidget implements IDashboardWidget,
 			let match = filePath.match(insertValueRegex);
 			if (match && match.length > 0 && match[1] === 'workspaceRoot') {
```

Comment:
hmm, do you know why we are using match[1] instead of match[0]? if 1 is correct, we should use match.length>1

Reply:
and in line 302, it is using match[0] again. very strange .

Reply:
Because of how the match is set up - they use groupings so match[0] refers to the whole macro string ${workspaceRoot} while match[1] will be just the inner part match[1].

It's unnecessary IMO - just checking match[0] === "${workspaceRoot}" would work fine. But it's also not broken so I'm fine just leaving it unless people think it's worth changing.